tests/main/preseed/task.yaml

```diff
@@ -97,16 +92,16 @@ execute: |
   [ "$(grep -c ' Done ' tasks.log)" = "32" ]
 
   # everything below is pending execution on first boot
-  MATCH "Do .+ run-hook +Run install hook of \"$core_snap\" snap if present" < tasks.log
+  MATCH "Do .+ run-hook +Run install hook of \"${LXD_BASE_SNAP}\" snap if present" < tasks.log
   MATCH "Do .+ start-snap-services +Start snap \"lxd\" \([0-9]+\) services" < tasks.log
   MATCH "Do .+ run-hook +Run configure hook of \"lxd\" snap if present" < tasks.log
   MATCH "Do .+ run-hook +Run health check of \"lxd\" snap" < tasks.log
   MATCH "Do .+ mark-seeded +Mark system seeded" < tasks.log
   MATCH "Do .+ run-hook +Run install hook of \"snapd\" snap if present" < tasks.log
   MATCH "Do .+ start-snap-services +Start snap \"snapd\" \(unset\) services" < tasks.log
   MATCH "Do .+ run-hook +Run configure hook of \"core\" snap if present" < tasks.log
```

Comment:
this one too?

Reply:
hmm, acutally I need to check, depends on if the core snap is included in the image we preseed manually or not, from the failure log I saw the snaps included in the image are:
* lxd
* lxd's base snap (core20 in this case)
* core snap
* snapd snap

I'm not sure why the core snap is there tbh

Reply:
oh huh actually there is no core snap in there, I think the description of this task is just wrong